src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -40,22 +71,255 @@
+                    </define-field>
+                    <define-field name="short-name">
+                        <formal-name>Party Short Name</formal-name>
+                        <description>A short common name, abbreviation, or acronym for the party.</description>
+                    </define-field>
+                    <define-field name="external-id" max-occurs="unbounded">
+                        <!-- This is an id because the idenfier is assigned and managed externally by humans. -->
+                        <formal-name>Party External Identifier</formal-name>
+                        <description>An identifier for a person or organization using a designated scheme. e.g. an Open Researcher and Contributor ID (ORCID)</description>
+                        <json-value-key>id</json-value-key>
+                        <group-as name="external-ids" in-json="ARRAY"/>
+                        <define-flag name="scheme" as-type="uri" required="yes">
+                            <formal-name>External Identifier Schema</formal-name>
+                            <description>Indicates the type of external identifier.</description>
+                            <constraint>
+                                <allowed-values allow-other="yes">
+                                    <enum value="http://orcid.org/">The identifier is Open Researcher and Contributor ID (ORCID).</enum>
+                                </allowed-values>
+                            </constraint>
+                        </define-flag>
+                    </define-field>
+                    <assembly ref="property" max-occurs="unbounded">
+                        <group-as name="props" in-json="ARRAY"/>
+                    </assembly>
+                    <assembly ref="link" max-occurs="unbounded">
+                        <group-as name="links" in-json="ARRAY"/>
+                    </assembly>
+                    <field ref="email-address" max-occurs="unbounded">
+                        <group-as name="email-addresses" in-json="ARRAY"/>
+                        <remarks>
+                            <p>This is a contact email associated with the party.</p>
+                        </remarks>
+                    </field>
+                    <field ref="telephone-number" max-occurs="unbounded">
+                        <group-as name="telephone-numbers" in-json="ARRAY"/>
+                        <remarks>
+                            <p>A phone number used to contact the party.</p>
+                        </remarks>
+                    </field>
+                    <choice>
+                        <assembly ref="address" max-occurs="unbounded">
+                            <group-as name="addresses" in-json="ARRAY"/>
+                        </assembly>
+                        <field ref="location-uuid" max-occurs="unbounded">
+                            <group-as name="location-uuids" in-json="ARRAY"/>
+                        </field>
```

Comment:
This choice allows for either to be provided or none. A remark should be added to this effect.

Reply:
Addressed this manually.